secator/utils.py

```
@@ -145,6 +146,96 @@ def deduplicate(array, attr=None):
+
+	Args:
+		package_dir (str): Package dir, relative to secator/ directory.
+		attr_name (str): Attribute name that needs to be present in the class.
+		base_class (str, Optional): Base class that our matches need to inherit from.
+
+	Returns:
+		list: List of matching classes.
+	"""
+	package_dir = Path(__file__).resolve().parent / package_dir
+	classes = []
+	for (_, module_name, _) in iter_modules([str(package_dir)]):
```

Ruff (0.11.9)

Undefined name `iter_modules`

(F821)